test/integration/job/job_test.go

```diff
@@ -2507,7 +2507,7 @@ func TestNodeSelectorUpdate(t *testing.T) {
 	// (2) Check that the pod was created using the expected node selector.
 
 	var pod *v1.Pod
-	if err := wait.PollImmediate(waitInterval, wait.ForeverTestTimeout, func() (bool, error) {
```

Comment:
Why the change from `err := ` to `err =` ?

Reply:
ditto.

Reply:
Done.